src/detext/model/sp_emb_model.py

```diff
@@ -26,25 +28,27 @@ def __init__(self,
             wide_ftrs_sp_val = tf.ones(tf.shape(wide_ftrs_sp_idx), dtype=tf.float32)
 
         self._num_wide_sp = num_wide_sp
+        self._padding_idx = padding_idx
 
         with tf.variable_scope('wide', reuse=tf.AUTO_REUSE):
             # Feature weights
             self.ftrs_weight = tf.get_variable('wide_ftrs_sp_weight',
-                                               shape=[num_wide_sp, 1],
+                                               shape=[num_wide_sp, sp_emb_size],
                                                initializer=initializer,
                                                trainable=True)
 
             # A hack to combine idx and val so that we can process them together in `tf.map_fn` later
+            # Shape=[batch_size, max_group_size, max_wide_ftrs_size*2]
```

Comment:
Can you comment on what the `max_wide_ftrs_size` is? I think I saw related comments somewhere else but it's great to consolidate them under this class.

Reply:
Added comment in this line